@lyra/api/external.ts

```diff
@@ -1,15 +1,15 @@
-import { EXTERNAL_SERVICE_TYPES } from "constants/services";
-import { sendMessageToContentScript } from "api/helpers";
-import { ExternalRequest } from "api/types";
+import { EXTERNAL_SERVICE_TYPES } from "@lyra/constants/services";
```

Comment:
`@lyra` is a folder for shared resources that the extension/API both need

Reply:
note about `@lyra/constants` and `@lyra/statics`: 

the folder structure dumps all the files plus the package.json/tsconfig.json into the root. I spent a lot of time trying different approaches to this, but settled on this bc:

- i wanted to import files directly by path, rather than having one `main` file to import from 
```
import { services } from '@lyra/constants'; 
const {EXTERNAL_SERVICE_TYPES} = services;
// gross
```
- related to the above, I didn't want to have to compile each workspace every time a change was made. I just wanted this "npm package" to be treated as a regular folder and have the parent application compile it
- I wanted to import using a clean path like `@lyra/constants/services` rather than `@lyra/constants/src/services`

Conditional exports (https://nodejs.org/api/esm.html#esm_conditional_exports) would solve the issue of having to dump everything in root, but I could not get it to work for some reason 🤔 Will continue to look into it